tests/test_qaoa_objective_maxcut.py

```diff
@@ -58,7 +58,7 @@ def test_maxcut_qaoa_obj_fixed_angles(simulator):
             for p in range(1, max_p + 1):
                 gamma, beta, AR = get_fixed_gamma_beta(d, p, return_AR=True)
                 f_e = get_qaoa_maxcut_objective(N, p, G=G, parameterization="gamma beta", simulator=simulator, objective="expectation")
-                f_o = get_qaoa_maxcut_objective(N, p, G=G, parameterization="gamma beta", simulator=simulator, objective="overlap")
```

Comment:
@rajgane07 may I ask you to clarify the reason for this change?

Reply:
@rsln-s  - My apologies, I was testing the code by using various permutations of simulators to check they provide same answers and forgot to replace it. I will revert the change back,

Other thing in portfolio Optimization I deliberately added extra object instead of returning in one line just for easy readability purpose.